instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import CommentSection from '../CommentSection/CommentSection';
+// import dummyData from '../../dummy-data';
+
+// const commentArray = dummyData.map(postObject => {
+//   return postObject['comments'];
+// });
+
+class PostContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      comment: [],
+      likes: null,
+      filteredData: [] 
+    }
+  }
+
+  componentWillMount() {
+    this.setState({
+      data: this.props.dummyData,
+      comment: this.props.comment,
+      likes: this.props.likes,
+      filteredData: this.props.filteredData
+    })
+  }
+
+  componentWillReceiveProps(newProps) {
+    this.setState({ filteredData: newProps.filteredData} );
+  }
+  
+  handleLike(e) {
+    const container = e.target.parentNode.parentNode.parentNode;
```

Comment:
Line 35: This is a little bit unclear. I like the way you're thinking about it, but I wonder if it might be better to have another presentational component that displays the post (maybe called Post?). Then you could pass the index in as a prop, and pass in this method (using either `.bind()` or ES6 arrow syntax) to use within the component as an event listener. Let me know if you need clarification on what I'm saying.